precompiles/erc20/erc20.go

```diff
@@ -26,16 +26,17 @@ const (
 	// abiPath defines the path to the ERC-20 precompile ABI JSON file.
 	abiPath = "abi.json"
 
```

Comment:
Maybe we can leave a comment here explaining where this values came from?

Reply:
Good point, I've added a describing comment 👍